app/dashboard/views/alias_log.py

```diff
@@ -1,27 +1,32 @@
-from dataclasses import dataclass
-
-import arrow
-from flask import render_template, flash, redirect, url_for
+from flask import render_template, flash, redirect, url_for, abort
 from flask_login import login_required, current_user
 
 from app.dashboard.base import dashboard_bp
 from app.extensions import db
 from app.models import GenEmail, ForwardEmailLog, ForwardEmail
 
+LIMIT = 15
```

Comment:
to avoid other modules from using this constant, I think we could add `_` prefix to its name